# request fields
# global path
nav_msgs/Path global_plan
---
# response fields
bool successful